src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMNavigator.java

```diff
@@ -493,7 +493,7 @@ public void visitSources(SCMSourceObserver observer) throws IOException, Interru
             } else {
                 // Navigate the repositories of the repoOwner as a user
                 listener.getLogger().format("Looking up repositories of user %s%n", repoOwner);
-                request.withRepositories(bitbucket.getRepositories(UserRoleInRepository.OWNER));
+                request.withRepositories(bitbucket.getRepositories(UserRoleInRepository.ADMIN));
```

Comment:
^^^ I didn't get as far as doing this experiment before the PR was merged.  On that basis, @guimarca I suggest you propose a fix along these lines in the context of #450 rather than here, to keep the concerns separate.  Good luck!